src/components/ContactUserItem/index.tsx

```diff
@@ -136,6 +156,15 @@ const UserItem: FC<
             </RnText>
           </View>
         )}
+        {!!fromMissedCall && (
+          <RnText
+            normal
+            small
+            style={[styles.callCreatedAt, { marginRight: 11 }]}
+          >
```

Comment:
Use ternary operator instead of !! and && conditions.

Reply:
Not resolved yet. Use ternary operator instead of !! and && conditions for lines 138 to 167.